package.json

```
@@ -106,7 +106,7 @@
     "common-tags": "1.8.0",
     "console.table": "0.10.0",
     "cy-spok": "1.3.2",
-    "cypress": "11.2.0",
+    "cypress": "https://cdn.cypress.io/beta/npm/12.0.0/linux-x64/release/12.0.0-79ea4537649ec49b72286fc6ba48089a0b9ff879/cypress.tgz",
```

Wait until 12.0.0 is released to apply this suggestion.

```suggestion
    "cypress": "^12.0.0"
```